src/app/helptext/jails/jails-edit.ts

```diff
@@ -320,6 +323,9 @@ permission is only effective when <b>allow_mount</b> \
 is set and <b>enforce_statfs</b> is set to a value \
 lower than <i>2</i>.'),
 
+allow_mount_fusefs_placeholder: T('allow_mount_fusefs'),
+allow_mount_fusefs_tooltip: T(''),
```

Comment:
Set to allow privileged users inside the jail to mount and unmount the fusefs file system. The jail must have FreeBSD 12.0 or newer installed. This permission is only effective when allow_mount is set and and enforce_statfs is set to a value lower than 2.